packages/ui/react/src/components/Forms/Input/Input.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+
+import { ErrorMessage } from '../../Primitives/ErrorMessage'
+import { Container, InputText } from './Input.styled'
+import { Label } from 'src/components/Forms/Label/Label'
+import type { LabelProps } from 'src/components/Forms/Label/Label'
```

Comment:
Can you apply the suggestion below to avoid repeating Label?

```suggestion
import { Label } from 'src/components/Forms/Label'
import type { LabelProps } from 'src/components/Forms/Label'
```